fdbserver/workloads/UnitTests.actor.cpp

```diff
@@ -153,8 +158,19 @@ struct UnitTestWorkload : TestWorkload {
 	}
 
 	bool testMatched(std::string const& testName) const {
-		return StringRef(testName).startsWith(testPattern) &&
-		       (!testsIgnored.present() || !StringRef(testName).startsWith(testsIgnored.get()));
+		if (!StringRef(testName).startsWith(testPattern)) {
+			return false;
+		}
+
+		bool matched = true;
+		for (auto ignorePatt : testsIgnored) {
+			if (StringRef(testName).startsWith(ignorePatt)) {
+				matched = false;
+				break;
+			}
+		}
+
+		return matched;
```

Comment:
Nit: slightly more readable i.e. two blocks of early `return false`, if we pass those then `return true`. 

```suggestion
		for (auto ignorePatt : testsIgnored) {
			if (StringRef(testName).startsWith(ignorePatt)) {
				return false;
			}
		}

		return true;
```